gcp/api/server.py

```diff
@@ -690,9 +691,19 @@ def serve(port: int, local: bool):
     server.stop(_SHUTDOWN_GRACE_DURATION)
 
 
+def is_cloud_run() -> bool:
+  """Check if we are running in Cloud Run."""
+  # https://cloud.google.com/run/docs/container-contract#env-vars
+  return os.getenv('K_SERVICE') is not None
+
+
 def main():
   """Entrypoint."""
-  logging.basicConfig(stream=sys.stderr)
+  if is_cloud_run():
+    logging_client = google.cloud.logging.Client()
+    logging_client.setup_logging()
```

Comment:
We could use https://github.com/google/osv.dev/blob/master/osv/logs.py which also surfaces error logs (without stack traces) to the error reporting dashboard.

Reply:
Good point! Imported that function to use instead.